## If there was an error with the invocation
## there might have been partial results
#if($ctx.error)
    ## Append a GraphQL error for that field in the GraphQL response
    $utils.appendError($ctx.error.message, $ctx.error.message)
#end
## Also returns data for the field in the GraphQL response
$util.toJson({
    "accountId": "${ctx.result.id}",
    "name": "${ctx.result.itemData.name}",
    "owner": "${ctx.result.owner}",
    "planId": "${ctx.result.itemData.planId}",
    "createdAt": "${ctx.result.createdAt}",
    "updatedAt": "${ctx.result.updatedAt}"
})